src/components/SortableCommentList.js

```diff
@@ -0,0 +1,207 @@
+    if (sectionComments && sectionComments.count !== sectionComments.results.length) {
+      setTimeout(() => this.fetchMoreComments(), 1000);
+      this.setState({showLoader: true});
+    }
+  }
+
+  render() {
+    const {
+      displayVisualization,
+      fetchComments,
+      // postComment,
+      section,
+      sectionComments,
+      canVote,
+      // voteComment,
+      ...rest
+    } = this.props;
+
+    const showCommentList = section && sectionComments && get(sectionComments, 'results');
+    return (
+      <div className="sortable-comment-list">
+        <h2><FormattedMessage id="comments"/>
+          <div className="commenticon">
+            <Icon name="comment-o"/>&nbsp;{get(sectionComments, 'count') ? sectionComments.count : ''}
+          </div>
+        </h2>
+        {
+          (typeof window !== 'undefined') && showCommentList && displayVisualization && get(section, 'plugin_identifier') === 'mapdon-ksv' ?
+            <div className="comments-visualization">
+              <MapdonKSVPlugin
+                data={section.plugin_data}
+                pluginPurpose="viewComments"
+                comments={sectionComments.results}
+              />
+              <div className="image-caption">Kaikki annetut kommentit sekä siirretyt ja lisätyt asemat kartalla.</div>
+              <MapdonKSVPlugin
+                data={section.plugin_data}
+                pluginPurpose="viewHeatmap"
+                comments={sectionComments.results}
+              />
+              <div className="image-caption">Siirrettyjen ja lisättyjen asemien tiheyskartta.</div>
+            </div>
+            : null
+        }
+        <form className="sort-selector">
+          <FormGroup controlId="sort-select">
+            <ControlLabel><FormattedMessage id="commentOrder"/></ControlLabel>
+            <FormControl componentClass="select" onChange={(event) => { console.log(event.target.value); fetchComments(section.id, event.target.value); }}>
+              {keys(ORDERING_CRITERIA).map((key) =>
+                <option key={key} value={ORDERING_CRITERIA[key]} selected={ORDERING_CRITERIA[key] === get(sectionComments, 'ordering')}>
+                  <FormattedMessage id={key}/>
+                </option>)
+              }
+            </FormControl>
+          </FormGroup>
+        </form>
+
+        { showCommentList &&
+          <div>
+            <CommentList
+              canVote={canVote}
+              section={section}
+              comments={sectionComments.results}
+              totalCount={sectionComments.count}
+              onPostComment={this.onPostHearingComment.bind(this)}
+              onPostVote={this.onVoteComment.bind(this)}
+              isLoading={this.state.showLoader}
+              {...rest}
+            />
+            <p className="sortable-comment-list__count">
+              <FormattedMessage id="commentsOutOf" values={{current: sectionComments.results.length, total: sectionComments.count }}/>
```

Comment:
Here, scrolling to the bottom increases the total count beyond those given in the section. 
![image](https://cloud.githubusercontent.com/assets/9367712/21567451/e0c1371a-ceb4-11e6-8f5f-6e3274510e60.png)